pkgs/by-name/pr/prmers/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  opencl-headers,
+  ocl-icd,
+  curl,
+  python3,
+  gmp,
+  libgnurl,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "prmers";
+  version = "4.15.35-alpha";
+
+  src = fetchFromGitHub {
+    owner = "cherubrock-seb";
+    repo = "PrMers";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-LAbWyz1FT9Zza4gewgKeMvQt8rwkfBhHRWBGpN+DIXE=";
+  };
+
+  enableParallelBuilding = true;
+  buildInputs = [
+    opencl-headers
+    ocl-icd
+    gmp
+    curl
+    python3
+    libgnurl
+  ];
+
+  installPhase = ''
```

Comment:
Why not use the upstream build target: https://github.com/cherubrock-seb/PrMers/blob/fbd35036d01d5a9b1495a774edc04e8677a4a253/Makefile#L62?